src/docs/deployment/web-renderers.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Web renderers
+description: How to choose a web renderer for running and building a web app.
+---
+
+Flutter provides two ways to display an application on the web:
+
+* **HTML**: Uses a combination of HTML elements, CSS, Canvas elements, and SVG
+  elements. This renderer has a smaller download size, 200KB for a simple app.
+* **CanvasKit:** Uses Skia compiled to WebAssembly and rendered to WebGL. This
+  renderer is fully consistent with Flutter mobile and desktop, has faster
+  performance, and is less likely to have differences across browsers, but adds
+  about 2MB in download size.
+
+## Command line flags
+
+The `--web-renderer` command line option takes one of three values, `html`,
+`canvaskit`, or `auto`.
+
+Choose the **html** renderer when optimizing for code size over performance on
+desktop and mobile devices.
+
+Choose the **canvaskit** renderer for applications that prioritize performance
+and pixel-perfect consistency
+
+Choose the **auto** option (default) for applications that are accessed from
```

Comment:
the wording of this confusing to me with how many times the word "accessed" is used and the "but" doesn't make sense.

it doesn't make it clear to me when I read it that auto will cause the app to use the HTML renderer on mobile devices, and CanvasKit on the desktop

Reply:
Maybe it's clearer to say "Choose the **auto** option (default) to switch between html on mobile devices and canvaskit on desktop devices." WDYT?

Reply:
I'm also confused.